ecs-agent/daemonimages/csidriver/driver/node_linux.go

```diff
@@ -0,0 +1,50 @@
+//go:build linux
+// +build linux
+
+// Copyright Amazon.com Inc. or its affiliates. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License"). You may
+// not use this file except in compliance with the License. A copy of the
+// License is located at
+//
+//	http://aws.amazon.com/apache2.0/
+//
+// or in the "license" file accompanying this file. This file is distributed
+// on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+// express or implied. See the License for the specific language governing
+// permissions and limitations under the License.
+
+package driver
+
+import (
+	"fmt"
+	"strconv"
+	"strings"
+
+	"golang.org/x/sys/unix"
+)
+
+// IsBlock checks if the given path is a block device
+func (d *nodeService) IsBlockDevice(fullPath string) (bool, error) {
```

Comment:
Why does this function need to be a method of `nodeService`? It doesn't use `nodeService` anywhere.

Reply:
Correct. But given it's used by nodeservice only and not used by many other components, so for now I think it's good to leave it here. But potentially it can be one util method.

Reply:
Ok I can change this to a util method.